Valet/VALValet.m

```diff
@@ -450,6 +450,10 @@ - (BOOL)setObject:(nonnull NSData *)value forKey:(nonnull NSString *)key options
             keychainData[(__bridge id)kSecValueData] = value;
             
             status = VALAtomicSecItemAdd((__bridge CFDictionaryRef)keychainData, NULL);
+            
+            if (status == errSecMissingEntitlement) {
```

Comment:
Maybe `VALCheckCondition(status != errSecMissingEntitlement, , A 'Missing Entitlements' error occurred. This is likely due to an Apple Keychain bug. As a workaround try running on a device that is not attached to a debugger.\n\nMore information: https://forums.developer.apple.com/thread/4743)`?

Curious, were you hitting #4 in https://github.com/square/Valet/blob/master/README.md#debugging? Might also be worth giving a shout out to that part of the README.

Thoughts on also logging this in `objectForKey:options:`?